newPassword.php

```diff
@@ -0,0 +1,40 @@
+<?php
+session_start();
+$email = $_POST["email"];
+$dob = $_POST["dob"];
+
+include 'connection.php';
+$sql = "SELECT * FROM users WHERE email=$email";
+$result = $connection->query($sql);
+
+if ($row = $result->fetch_assoc) {
+  if ($dob == $row["dob"]) {
+    $_SESSION["forgotPasswordEmail"] = $email;
+    header('Location: ./index.php?alert=Your Password has been reset');
```

Comment:
You don't need to redirect anywhere, if the `dob` matches, then just let the user continue on this page. And fill out his/her new password.